README.md

```diff
@@ -75,7 +81,7 @@ logger.Warn("A warning message");
 logger.Error("An error message");
 ```
 
-# System.Diagnostics
+## System.Diagnostics
```

Comment:
@SergeyKanzhelev I couldn't find a webpage to share for users to learn more information about System.Diagnostics or how they could fully use the TraceListener. Do you have any url that's relevant to share here?

Reply:
This one? https://docs.microsoft.com/en-us/dotnet/framework/debug-trace-profile/tracing-and-instrumenting-applications